ddtrace.gemspec

```diff
@@ -35,6 +35,7 @@ EOS
   spec.add_dependency 'msgpack'
 
   spec.add_development_dependency 'rake', '~> 10.5'
+  spec.add_development_dependency 'excon', '~> 0.62.0'
```

Comment:
I don't think we should add this here. Excon is considered a "contrib", and we load these kind of external dependencies via Appraisals, which are used to drive tests. Development dependencies are for the gem itself.